doc/changelogs/CHANGELOG_V14.md

```diff
@@ -71,6 +72,36 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="14.19.2"></a>
+## 2022-04-29, Version 14.19.2 'Fermium' (LTS), @juanarbol
```

Comment:
We tend to avoid Friday releases - would 2022-05-03 work for you?
Also we'll need to have someone else promote this time as your key isn't available yet. (I should be around on Tuesday, if that date does happen to work?)

Reply:
Sure! I will replace the date for `2022-05-03`

Reply:
Fixed!